docs/arch_services.md

```diff
@@ -0,0 +1,52 @@
+# OpenVoiceOS Services
+** Editors Note **  
+Major revisions coming here, mostly placeholder information
+
+## Skills Service
+
+The skills service is responsible for loading skills and intent parsers
+
+All user queries are handled by the skills service, you can think of it as OVOS's brain
+
+## Speech Service
+### Speech Client
+
+The speech client is responsible for loading STT, VAD and Wake Word plugins
+
+Speech is transcribed into text and forwarded to the skills service
+
+### Hotwords
+
+OVOS allows you to load any number of hot words in parallel and trigger different actions when they are detected
+
+each hotword can do one or more of the following:
+
+- trigger listening, also called a **wake_word**
+- play a sound
+- emit a bus event
+- take ovos-core out of sleep mode, also called a **wakeup_word** or **standup_word**
+- take ovos-core out of recording mode, also called a **stop_word**
+
+To add a new hotword add its configuration under "hotwords" section.
+
+By default, all hotwords are disabled unless you set `"listen": true`. 
+Under the `"listener"` setting a main wake word and stand up word are defined, those will be automatically enabled unless you set `"active": false`. 
```

Comment:
I think this still doesn't make a lot of sense as to what is needed.  Are they all enabled to listen unless `"active": false` is set, or are they all disabled unless `"listen": true` is set?

Reply:
by default all wake words are `"active": false`, this means they won't be loaded, the ww you select as main wake word under "listener" section will default to `true` instead

in practice this means you only have 1 wake word at a time loaded, unless you explicitly set `"active": true` to enable multiple ones at same time

the other settings (listen/sound/utterance/stt_lang....) are modifiers for what happens when the ww is detected